.github/workflows/llm_performance_tests.yml

```diff
@@ -59,13 +60,22 @@ jobs:
           python -m pip install --upgrade transformers_stream_generator
           python -m pip install --upgrade tiktoken
 
-      - name: Download llm binary
-        uses: ./.github/actions/llm/download-llm-binary
+      # please uncomment it and comment the "Install BigDL-LLM from Pypi" part for PR tests
+      # - name: Download llm binary
+      #   uses: ./.github/actions/llm/download-llm-binary
 
-      - name: Run LLM install (all) test
-        uses: ./.github/actions/llm/setup-llm-env
-        with:
-          extra-dependency: "xpu_2.1"
+      # - name: Run LLM install (all) test
+      #   uses: ./.github/actions/llm/setup-llm-env
+
+      - name: Install BigDL-LLM from Pypi
+        shell: bash
+        run: |
+          pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+          test_version_data=`date -d 'yesterday' '+%Y%m%d'`
+          if ! pip show bigdl-llm | grep $test_version_data; then
+            echo "Did not install bigdl-llm with excepted version $test_version_data"
+            exit 1
```

Comment:
Maybe we put this as the last step of the test to make the job fail? If we fail to release on some day, it is still reasonable to run the version of the previous day?

Reply:
I think if we change to install from pypi, we may want to test bigdl-llm for each release version. If so, if the release job failed, the performance test of that release should also be meaningless? And it would be more clear for reproducing the performance results.